latest/root/usr/bin/run-configsvr

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source ${CONTAINER_SCRIPTS_PATH:-}/environment
+
+# shellcheck source=/dev/null
+source ${CONTAINER_SCRIPTS_PATH:-}/generate_container_user
+
+# shellcheck source=/dev/null
+source ${CONTAINER_SCRIPTS_PATH:-}/db/replset/configsvr/init-configsvr
+
+unset share_files share_file
+trap cleanup SIGINT SIGTERM
+
+log_info "Checking required env variables"
+check_repl_env_vars
+
+log_info "Checking data directory"
+check_data_dir
+
+log_info "Setting up storage cache"
+setup_wiredtiger_cache "${CONTAINER_SCRIPTS_PATH:-}/mongodb.conf.template"
+
+log_info "Setting up config file"
+if [[ ! -s "$MONGODB_CONFIG_PATH" ]]; then
+	envsubst < "${CONTAINER_SCRIPTS_PATH:-}/mongodb.conf.template" > "$MONGODB_CONFIG_PATH"
+fi
+
+log_info "Settings up keys"
+setup_keyfile
+
+log_info "Starting mongo configuration server"
+$MONGOD --config "$MONGODB_CONFIG_PATH" --configsvr --keyFile $MONGODB_KEYFILE_PATH --replSet "${MONGODB_REPLICA_NAME:-}" &
```

Comment:
`--configsvr` option is useless in our containers anyway. 

1. we change database location anyway
2. port does not matter in containers (no conflicting)

Reply:
Not sure I understand all of this comment. For number 2, correct, we don't have to worry about ports conflicting. I explicitly set the default 27017 in the configuration for this reason - because config server assumes running on the same host, and runs on a different port by default (27019)